testing/src/test/java/io/stargate/it/http/DocumentApiV2Test.java

```diff
@@ -2139,65 +2256,70 @@ public void deleteNamespace() throws IOException {
         HttpStatus.SC_NOT_FOUND);
   }
 
-  private Response get(String path) throws IOException {
-    Request request =
+  private Response get(String path, boolean authorize) throws IOException {
+    Request.Builder request =
         new Request.Builder()
             .url(String.format("%s:8082%s%s", host, path.startsWith("/") ? "" : "/", path))
-            .get()
-            .addHeader("X-Cassandra-Token", authToken)
-            .build();
+            .get();
+    if (authorize) {
+      request = request.addHeader("X-Cassandra-Token", authToken);
+    }
 
-    return client.newCall(request).execute();
+    return client.newCall(request.build()).execute();
   }
 
-  private Response post(String path, Object arg) throws IOException {
-    Request request =
+  private Response post(String path, Object arg, boolean authorize) throws IOException {
+    Request.Builder request =
         new Request.Builder()
             .url(String.format("%s:8082%s%s", host, path.startsWith("/") ? "" : "/", path))
             .post(
                 RequestBody.create(
-                    MediaType.parse("application/json"), objectMapper.writeValueAsString(arg)))
-            .addHeader("X-Cassandra-Token", authToken)
-            .build();
+                    MediaType.parse("application/json"), objectMapper.writeValueAsString(arg)));
+    if (authorize) {
+      request = request.addHeader("X-Cassandra-Token", authToken);
+    }
 
-    return client.newCall(request).execute();
+    return client.newCall(request.build()).execute();
   }
 
-  private Response put(String path, Object arg) throws IOException {
-    Request request =
+  private Response put(String path, Object arg, boolean authorize) throws IOException {
+    Request.Builder request =
         new Request.Builder()
             .url(String.format("%s:8082%s%s", host, path.startsWith("/") ? "" : "/", path))
             .put(
                 RequestBody.create(
-                    MediaType.parse("application/json"), objectMapper.writeValueAsString(arg)))
-            .addHeader("X-Cassandra-Token", authToken)
-            .build();
+                    MediaType.parse("application/json"), objectMapper.writeValueAsString(arg)));
+    if (authorize) {
+      request = request.addHeader("X-Cassandra-Token", authToken);
+    }
 
-    return client.newCall(request).execute();
+    return client.newCall(request.build()).execute();
   }
 
-  private Response patch(String path, Object arg) throws IOException {
-    Request request =
+  private Response patch(String path, Object arg, boolean authorize) throws IOException {
```

Comment:
Could we keep the old method signature as an overload to avoid the many change with `true` for the new parameter?

Reply:
yeaaa I was thinking about doing that

Will do :)

Reply:
I added an overloading for each with signature `String path, Object arg, String token`

So that I could test garbage tokens too (if it's null, the header will get left off)